### DISCUSSION TITLE
When will docker hub's latest-aio-gpu-nvidia-cuda-12 be updated?

### DISCUSSION THREAD

**Discussion by Taikono-Himazin** [mudler/LocalAI#1982] [Q&A]
When will docker hub's latest-aio-gpu-nvidia-cuda-12 be updated?
The latest version of v2.12.1 seems to have been updated, but it seems that the latest-aio-gpu-nvidia-cuda-12 has not been updated yet.
When will it be updated?
Also, please let me know if there is a way to build latest-aio-gpu-nvidia-cuda-12 at hand.

**Comment by mudler** [comment]
![Screenshot from 2024-04-10 09-33-36](https://github.com/mudler/LocalAI/assets/2420543/ae306b50-6e0c-435c-8eb2-bcb137c456fb)
looks all are up-to-date here, what shows when you curl `/version` in LocalAI?

**Comment by Taikono-Himazin** [comment]
![image](https://github.com/mudler/LocalAI/assets/25480502/88db2481-3dcc-4a65-91cb-c55627966643)
When I look at docker hub, it looks like it was updated 14 days ago.
Am I wrong here?
https://hub.docker.com/layers/localai/localai/latest-aio-gpu-nvidia-cuda-12/images/sha256-f32e743a00827a4291d3b0bf4bf82f14665e171c99d7d5e502876c4ae53d7858?context=explore

**Reply by mudler** [reply]
ouch - you are right, good catch. Seems something have failed when tagging the latest images on dockerhub!

**Reply by mudler** [reply]
images should be updated soon, thanks for raising this up!

**Reply by Taikono-Himazin** [reply]
thank you.
![image](https://github.com/mudler/LocalAI/assets/25480502/c934e808-b992-460a-a3cb-9b4df133b176)
